pages/getting_started.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Getting Started 
+order: 1
+---
+
+MDAnalysis is a powerful Python library for analyzing molecular dynamics simulations. This page will guide you through quickly installing MDAnalysis, exploring a basic example, and accessing learning resources to get started.
+
+## Installation Instructions
+
+For a quick start, check out our [Installation Quick Start][] guide. 
+For a more thorough installation with advanced options, including dependencies and environment configurations, see the [installation instructions in the User Guide]({{ site.docs.userguide.url }}/stable/installation.html). 
+
+Once installed, explore this [basic example][] to load a trajectory and perform a simple analysis.
```

Comment:
The example should not be under installation. Make it a separate heading – because that's how you advertised in the summary at the top.

I'd actually include the basic example here. It's nice to immediately see how to use MDAnalysis. The syntax-highlighted code is a good illustration!